src/cli/services/utils/utils.ts

```diff
@@ -19,7 +19,7 @@ ${chalk.bold('npm install -g npm@latest')}
 
 async function checkNodeVersion(session: ISession): Promise<boolean> {
   const checking = new Promise<boolean>((resolve) => {
-    check({ node: '>= 14.0.0 <17.0.0', npm: '>=6' }, (error, result) => {
+    check({ node: '>= 14.0.0', npm: '>=6' }, (error, result) => {
```

Comment:
Removed 17 restriction, as it seems to work fine on 18, and added that to the testing.

Reply:
See also #102